app/libtest.sh

```diff
@@ -16,9 +16,23 @@ APPDIR=${APPDIR:-$(pwd)}
 APP=${APP:-$(basename $APPDIR)}
 # Name of conode-log
 COLOG=conode
+# Base port when generating the configurations
```

Comment:
Are all of these changes just from a sync with cothority/libtest.sh?

Reply:
Yes, that's a verbatim copy of `cothority/libtest.sh`. Not sure how you want to handle this, but I needed `libtest.sh`, so instead of copying it in `onet/dbadmin`, I updated the `onet/app/libtest.sh`.